rest-api-spec/src/main/resources/rest-api-spec/test/cluster.health/10_basic.yml

```diff
@@ -92,6 +92,37 @@
   - match:     { unassigned_shards:       0 }
   - gte:       { number_of_pending_tasks: 0 }
 
+---
+"cluster health basic test, one index with wait for no initializing shards":
+  - skip:
+      version: " - 6.99.99"
+      reason: "wait_for_no_initializing_shard is introduced in 7.0.0"
+
+  - do:
+      indices.create:
+        index: test_index
+        wait_for_active_shards: 0
+        body:
+          settings:
+            index:
+              number_of_shards: 50
```

Comment:
The default of 5 is good enough (slow CI machines ftw)

Reply:
Yep, I had to increase it to see the difference on my machine.